src/scene/graphics/shared/__tests__/GraphicsContextSystem.test.ts

```diff
@@ -0,0 +1,104 @@
+import { GraphicsContext } from '../GraphicsContext';
+import { GraphicsContextSystem } from '../GraphicsContextSystem';
+import { getWebGLRenderer } from '@test-utils';
+
+describe('GraphicsContextSystem', () =>
+{
+    describe('ShapeBuildCommand', () =>
+    {
+        it('no skip', async () =>
```

Comment:
Can we make more descriptive text to describe what this test does rather than `no skip`? Same for the other's below. It's difficult to know what this is testing.